platform/linux-generic/odp_traffic_mngr.c

```diff
@@ -2607,6 +2607,10 @@ static int tm_capabilities(odp_tm_capabilities_t capabilities[],
 		per_level_cap->max_priority       = ODP_TM_MAX_PRIORITIES - 1;
 		per_level_cap->min_weight         = ODP_TM_MIN_SCHED_WEIGHT;
 		per_level_cap->max_weight         = ODP_TM_MAX_SCHED_WEIGHT;
+		per_level_cap->min_burst          = 1U;
```

Comment:
I wonder why min burst cannot be zero? Especially since this is a shaper and not policer?

Reply:
Made it zero for linux-gen